src/Http/Validation/Domain.php

```diff
@@ -53,4 +55,32 @@ protected function stripProtocol($domain)
 
         return $domain;
     }
+
+    /**
+     * Strip the port from a domain.
+     *
+     * @param $domain
+     *
+     * @return mixed
+     */
+    protected function stripPort($domain)
+    {
+        $domainStripped = preg_replace(self::PATTERN_STRIP_PROTOCOL, '', $domain);
```

Comment:
No in case of an error 

```php
if ($domainStripped = preg_replace(self::PATTERN_STRIP_PROTOCOL, null, $domain)) {
    return $domainStripped;
}

return $domain;
```

the return value would be the given domain, not null.